Dockerfile

```diff
@@ -1,9 +1,7 @@
 # ---- Base Image ----
-FROM node:12-alpine as base
-
-WORKDIR /opt/ccd-definition-processor
-
-COPY package.json yarn.lock ./
+FROM hmctspublic.azurecr.io/base/node/stretch-slim-lts-10:10-stretch-slim as base
```

Comment:
Please revert. Feature that split files into smaller chunks needs node 11 or newer

Reply:
Thanks, any idea why this docker cloud build either:

1) takes ages to build
2) if it does build successfully, it doesn't update the PR check status

https://cloud.docker.com/u/hmcts/repository/registry-1.docker.io/hmcts/ccd-definition-processor/builds/ef2b3c4b-c8e2-4d3b-9784-eeef1f38e5a2